<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="uiPages">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="ConnectionPage">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QLabel" name="labelIp">
            <property name="text">
             <string>IP :</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLineEdit" name="lineIp">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QLabel" name="labelPort">
            <property name="text">
             <string>Port:</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLineEdit" name="linePort">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="buttonConnect">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="labelFirstPage">
            <property name="font">
             <font>
              <family>URW Gothic L</family>
              <pointsize>16</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>Babel Project</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="LoginRegisterPage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_1">
          <item>
           <widget class="QLabel" name="labelLogo">
            <property name="font">
             <font>
              <family>URW Gothic L</family>
              <pointsize>12</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>BabelProject</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>12</number>
            </property>
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item alignment="Qt::AlignHCenter">
             <widget class="QLineEdit" name="lineUsername">
              <property name="font">
               <font>
                <family>URW Gothic L</family>
                <italic>true</italic>
               </font>
              </property>
              <property name="text">
               <string>username</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
             <widget class="QLineEdit" name="linePassword">
              <property name="font">
               <font>
                <family>URW Gothic L</family>
                <italic>true</italic>
               </font>
              </property>
              <property name="text">
               <string>password</string>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QRadioButton" name="radioButtonShowPassword">
              <property name="text">
               <string>Show password</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="labelUserDoesntExist">
              <property name="font">
               <font>
                <family>URW Gothic L</family>
                <italic>true</italic>
               </font>
              </property>
              <property name="text">
               <string>User doesn't exist !</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="labelUserWrongPassword">
              <property name="font">
               <font>
                <family>URW Gothic L</family>
                <italic>true</italic>
               </font>
              </property>
              <property name="text">
               <string>Wrong password.</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="labelUserIsAlreadyExist">
              <property name="font">
               <font>
                <family>URW Gothic L</family>
                <italic>true</italic>
               </font>
              </property>
              <property name="text">
               <string>This username is already in use.</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="labelUserIsAlreadyOn">
              <property name="font">
               <font>
                <family>URW Gothic L</family>
                <italic>true</italic>
               </font>
              </property>
              <property name="text">
               <string>This user is actually online</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QPushButton" name="buttonLogin">
              <property name="text">
               <string>Login</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QPushButton" name="buttonRegister">
              <property name="text">
               <string>Register</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="LoggedPage">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <widget class="QTableWidget" name="tableContactList">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <column>
               <property name="text">
                <string>Contact Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Online / Offline</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineAddContact">
              <property name="text">
               <string>new contact</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonAddContact">
              <property name="text">
               <string>Add new contact ?</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonLogOut">
              <property name="text">
               <string>Log Out</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item alignment="Qt::AlignHCenter|Qt::AlignTop">
             <widget class="QLabel" name="labelContactSelected">
              <property name="font">
               <font>
                <family>URW Gothic L</family>
                <pointsize>11</pointsize>
                <italic>true</italic>
               </font>
              </property>
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Select a Contact to Call</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter|Qt::AlignTop">
             <widget class="QLabel" name="labelContactAvatar">
              <property name="text">
               <string>Avatar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonLeaveCall">
              <property name="text">
               <string>leave call</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="buttonCall">
                <property name="minimumSize">
                 <size>
                  <width>180</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Call</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="1" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="labelWelcomeUserName">
          <property name="font">
           <font>
            <family>URW Gothic L</family>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>Weclome to you</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
